### PR TITLE
Hotfix/journalhosting lang bug

### PR DESCRIPTION
**Pull request by hmert**

### CODE REVIEW

File: src/Ojs/JournalBundle/Controller/CommonController.php [hunk 1]
```diff
@@ -4,20 +4,27 @@
...
+     * @return \Symfony\Component\HttpFoundation\RedirectResponse
+     */
     public function changeLocaleAction($code, Request $request)
     {
+        $response = new Response();
         $session = $this->get('session');
         $request->setLocale($code);
         $session->set('_locale', $code);
         $session->set('_locale_prefered', new \DateTime());
-        $referer = $request->headers->get('referer');
```
Comment (behram) [inline src/Ojs/JournalBundle/Controller/CommonController.php]: @hmert why did u removed this?

Comment (hmert) [inline src/Ojs/JournalBundle/Controller/CommonController.php]: just wait, i will push 1 commit

Comment (behram) [inline src/Ojs/JournalBundle/Controller/CommonController.php]: @hmert great you did my commit 😄

**Comment by behram** [comment]
i will merge  after travis build

**Comment by behram** [comment]
i reviewed all changes and looks like there is no problem

**Pull request merged**